test/e2e/resize_nodes.go

```diff
@@ -111,25 +112,26 @@ func waitForGroupSize(size int) error {
 	return fmt.Errorf("timeout waiting %v for node instance group size to be %d", timeout, size)
 }
 
-func svcByName(name string) *api.Service {
+func svcByName(name string, port int) *api.Service {
```

Comment:
Can we move these two functions to e2e/util now that they're being used across multiple tests.  They're definitely not specific to resize_nodes.